modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/repositories/AccountsRepository.java

```diff
@@ -55,7 +55,7 @@ public AccountsRepository(Ignite ignite, TransactionManager txMgr) {
         this.txMgr = txMgr;
 
         accountsTbl = new Table<Account>(ignite, "accounts")
-            .addUniqueIndex(Account::getUsername, (acc) -> "Account with email '" + acc.getUsername() + "' already registered")
+            .addUniqueIndex(a -> a.getUsername().trim().toLowerCase(), (acc) -> "Account with email '" + acc.getUsername() + "' already registered")
```

Comment:
I think it make sense to introduce function "normalizeEmail(string)" and use this function not only here (in index), but also when we save account.

Reply:
This logic is used only in index.